airbyte-cdk/python/README.md

```diff
@@ -99,6 +106,7 @@ pip install -e .
 ```
 
 Then, navigate to the CDK and install it in editable mode:
+
 ```bash
 cd ../../../airbyte-cdk/python
 pip install -e .
```

Comment:
Will this still work? It's a pretty important workflow (at least I'm using it daily), so we should make sure it works with Poetry in the CDK and pip in the connector, at least for a transition period

Reply:
Yep! `pip install -e .` should still work with Poetry packages. 👍